### ISSUE TITLE
Diacritics in file name makes data-system crash

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-data-system#238]
A user is trying ti upload to Files a huge file (160 Mo). Upload fails, Files returns a 500 error, after loosing connection with the data-system. Seems like the Ds crashes during upload. Here are the log:
```
[Sun, 12 Jun 2016 10:10:32 GMT] - POST /data/ - 201 - 3.590 ms -  42
[2016-06-12 12:10:32:383] info - helper/db_feed | Publishing usage.application files
[2016-06-12 12:10:32:384] info - helper/db_feed | Publishing file.create e138a4b8fc981b613175fc611aef1897
[2016-06-12 12:10:32:385] info - indexer | add batch 1 of 1 file
[2016-06-12 12:10:32:414] info - helper/db_feed | Publishing usage.application files
[2016-06-12 12:10:32:417] info - helper/db_feed | Publishing usage.application files
[Sun, 12 Jun 2016 10:10:32 GMT] - GET /data/e138a4b8fc981b613175fc611aef1897/ - 200 - 25.744 ms -  278
info - indexcontroller | app used deprecated POST /data/index/:id
[Sun, 12 Jun 2016 10:10:32 GMT] - POST /data/index/e138a4b8fc981b613175fc611aef1897 - 200 - 0.384 ms -  16
_http_outgoing.js:351
      throw new TypeError('The header content contains invalid characters');
      ^

TypeError: The header content contains invalid characters
    at ClientRequest.OutgoingMessage.setHeader (_http_outgoing.js:351:13)
    at new ClientRequest (_http_client.js:79:14)
    at Object.exports.request (http.js:31:10)
    at Request.start (/usr/local/cozy/apps/data-system/node_modules/request/request.js:746:30)
    at Request.write (/usr/local/cozy/apps/data-system/node_modules/request/request.js:1345:10)
    at PassThrough.ondata (/usr/local/cozy/apps/data-system/node_modules/multiparty/node_modules/readable-stream/lib/_stream_readable.js:572:20)
    at emitOne (events.js:77:13)
    at PassThrough.emit (events.js:169:7)
    at PassThrough.Readable.read (/usr/local/cozy/apps/data-system/node_modules/multiparty/node_modules/readable-stream/lib/_stream_readable.js:405:10)
    at flow (/usr/local/cozy/apps/data-system/node_modules/multiparty/node_modules/readable-stream/lib/_stream_readable.js:781:26)
```

**Comment by clochix** [comment]
Looks like its not due to the size of the file, but to its name: upload of `théâtre` fails, upload of `theatre` works.

**Comment by ook** [comment]
I confirm, the size is not the problem. You can use the attached file to reproduce. If you rename the file `piou.txt` it will just works.
[piou.txt](https://github.com/cozy/cozy-data-system/files/310731/piou.txt)
[theâtre.txt](https://github.com/cozy/cozy-data-system/files/310732/theatre.txt)

**Comment by pierrevdk** [comment]
I have the same problem with a .png file of 145Ko.
Capture d’écran 2016-06-13 à 11.01.31.png doesn't work
Neither does Capture d’ecran 2016-06-13 à 11.01.31.png
but pomme.png works
(of course this is the same file)

**Comment by poupotte** [comment]
This issue should be fixed with new version (2.5.10)
